controllers/drplacementcontrol_controller.go

```diff
@@ -793,6 +803,12 @@ func (d *DRPCInstance) runInitialDeployment() (bool, error) {
 		return done, nil
 	}
 
+	return d.startDeploying(homeCluster, homeClusterNamespace)
+}
+
+func (d *DRPCInstance) startDeploying(homeCluster, homeClusterNamespace string) (bool, error) {
```

Comment:
(nit) rename this to `depoyToHomeCluster` instead. Keeping it as,
`runFailover` -> `switchToFailoverCluster`
`runInitialDeployment` -> `deployToHomeCluster` and so on...